app/views/caseload.html

```diff
@@ -0,0 +1,58 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+{% include "includes/breadcrumbs.html" %}
+
+{% include "includes/validation-error-messages.html" %}
+
+{% include "includes/title-subtitle.html" %}
+
+{% include "includes/sub_nav.html" %}
+
+<h3 class="heading-medium">Overall caseload by tier</h3>
+
+<table class="sln-table-caseload data-table">
```

Comment:
Was wondering about this. Only overview is separated out into includes, case progress and capacity are embedded.